### PR TITLE
fix: Block Action execution for Post UQI datasources like oracle

### LINKED ISSUE #24350: [Bug]: Orcale Run button is inactive even after adding data
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When the user adds appropriate creds and tries to run the query it is observed that RUN button is inactive

### Steps To Reproduce

1) Add the connection details 
2) Save the datasource
3) Click on Add query 
4) Select the command 
and observe the RUN button is not active 
Screenshot 2023-06-12 at 11 36 10 AM

### Public Sample App

_No response_

### Environment

Release

### Issue video log

_No response_

### Version

Cloud

### PR DESCRIPTION
**Pull request by Irongade**
The configProperty path for Post UQI datasource's body field  is stored in actionConfiguration.formData.body.data, other older pre UQI format SQL datasources  actionConfiguration.body. This adds a check for such post UQI datasources like Oracle.

Fixes #24350 

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Chore (housekeeping or task changes that don't impact user perception)
- This change requires a documentation update
>
>
>
- [ ] Manual
- [ ] Jest
- [ ] Cypress
>
>
#### Test Plan
Tested the following for Oracle DS - 

1. Created oracle DS - and from review page, using new query button, created select/insert/update/delete queries and checked for run button functionality
2. From the schema of oracle DS, created select/insert/update/delete queries and checked for run button functionality
3. From the New Query add button on the Entity explorer, created select/insert/update/delete queries and checked for run button functionality>
4. Quick tested PostGreSQL query run, it is rightly disabled if query field is blank, and if anything entered into the query field, run gets enabled
5. Quick tested Snowflake query run, here noted that even if the query field is blank, the run button is enabled.

#### Dev activity
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

#### QA activity:
- [ ] [Speedbreak features](https://github.com/appsmithorg/TestSmith/wiki/Test-plan-implementation#speedbreaker-features-to-consider-for-every-change) have been covered
- [ ] Test plan covers all impacted features and [areas of interest](https://github.com/appsmithorg/TestSmith/wiki/Guidelines-for-test-plans/_edit#areas-of-interest)
- [ ] Test plan has been peer reviewed by project stakeholders and other QA members
- [x] Manually tested functionality on DP
- [ ] We had an implementation alignment call with stakeholders post QA Round 2
- [ ] Cypress test cases have been added and approved by SDET/manual QA
- [ ] Added `Test Plan Approved` label after Cypress tests were reviewed
- [ ] Added `Test Plan Approved` label after JUnit tests were reviewed

### CODE REVIEW

File: app/client/cypress/e2e/Regression/ServerSide/ActionExecution/Block_Execution.ts [hunk 1]
```diff
@@ -30,4 +30,20 @@ describe("Block Action Execution when no field is present", () => {
...
   });
+
+  it("3. Ensure Run button is disabled for Post UQI Datasources e.g. Oracle when no body data is present", () => {
+    let name: any;
+    dataSources.CreateDataSource("Oracle", true, false);
+    cy.get("@dsName").then(($dsName) => {
+      name = $dsName;
+
+      agHelper.Sleep(1000);
+      dataSources.NavigateFromActiveDS(name, true);
```
Comment (Aishwarya-U-R) [inline app/client/cypress/e2e/Regression/ServerSide/ActionExecution/Block_Execution.ts:41]: nit: `CreateQueryAfterDSSaved` can be used if trying to create query from the freshly saved DS. Need not navigate again from Active tab

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress looks OK!

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress looks ok!

**Review by sneha122 — DISMISSED** [review]

**Comment by btsgh** [comment]
As per discussion with @Irongade - he will provide a quick fix for snowflake as well and that can go into this same PR.

**Review by sneha122 — APPROVED** [review]

**Pull request merged**